docs/setup/install/deb.asciidoc

```diff
@@ -99,6 +99,35 @@ Examine +/etc/apt/sources.list.d/kibana-{major-version}.list+ for the duplicate
 
 endif::[]
 
+ifeval::["{release-state}"!="unreleased"]
+
+[NOTE]
+==================================================
+
+An alternative package which contains only features that are available under the
```

Comment:
thoughts on adding `An alternative package, `kibana-oss`, ...?  I had to search for the package name when running through the instructions.

Reply:
Good idea, added in https://github.com/elastic/kibana/pull/26475/commits/532383d635a76ae7b8077206cfadb36e9662298e.